common.gypi

```diff
@@ -75,8 +75,16 @@
 
     'v8_win64_unwinding_info': 1,
 
-    # TODO(refack): make v8-perfetto happen
+    # Variables controlling external defines exposed in public headers.
+    'v8_enable_conservative_stack_scanning%': 0,
+    'v8_enable_direct_local%': 0,
+    'v8_enable_map_packing%': 0,
+    'v8_enable_pointer_compression_shared_cage%': 0,
+    'v8_enable_sandbox%': 0,
+    'v8_enable_v8_checks%': 0,
+    'v8_enable_zone_compression%': 0,
     'v8_use_perfetto': 0,
+    'tsan%': 0,
```

Comment:
Why this not prefix with v8 ?

Reply:
This is what is used in `tools/v8_gypfiles/toolchain.gypi`, and `asan` does not have a v8 prefix too.